src/PKSim.Infrastructure/Services/ImportObservedDataTask.cs

```diff
@@ -106,78 +93,42 @@ private void adjustMolWeight(DataRepository observedData)
          _parameterChangeUpdater.UpdateMolWeightIn(observedData);
       }
 
-      private IReadOnlyList<ColumnInfo> fractionImportConfiguration()
-      {
-         var columns = new List<ColumnInfo>();
-         var timeColumn = createTimeColumn();
-         columns.Add(timeColumn);
-
-         var fractionInfo = new ColumnInfo
-         {
-            DefaultDimension = _dimensionRepository.Fraction,
-            Name = PKSimConstants.UI.Fraction,
-            IsMandatory = true,
-            NullValuesHandling = NullValuesHandlingType.DeleteRow,
-            BaseGridName = timeColumn.Name
-         };
-         fractionInfo.DimensionInfos.Add(new DimensionInfo {Dimension = _dimensionRepository.Fraction, IsMainDimension = true});
-         columns.Add(fractionInfo);
-         return columns;
-      }
 
-      private IReadOnlyList<ColumnInfo> amountImportConfiguration()
+      private IReadOnlyList<ColumnInfo> importConfiguration()
       {
          var columns = new List<ColumnInfo>();
          var timeColumn = createTimeColumn();
          columns.Add(timeColumn);
 
-         var amountInfo = new ColumnInfo
-         {
-            DefaultDimension = _dimensionRepository.Amount,
-            Name = PKSimConstants.UI.Amount,
-            IsMandatory = true,
-            NullValuesHandling = NullValuesHandlingType.DeleteRow,
-            BaseGridName = timeColumn.Name
-         };
-
-         amountInfo.DimensionInfos.Add(new DimensionInfo {Dimension = _dimensionRepository.Amount, IsMainDimension = true});
-         amountInfo.DimensionInfos.Add(new DimensionInfo {Dimension = _dimensionRepository.Mass, IsMainDimension = false});
-         columns.Add(amountInfo);
-
-         var errorInfo = createErrorColumnInfo(amountInfo, _dimensionRepository.Mass, _dimensionRepository.NoDimension);
-         columns.Add(errorInfo);
-
-         return columns;
-      }
 
-      private IReadOnlyList<ColumnInfo> concentrationImportConfiguration()
-      {
-         var columns = new List<ColumnInfo>();
-         var timeColumn = createTimeColumn();
-         columns.Add(timeColumn);
+         var supportedDimensions = new[] {
```

Comment:
MoBi allows everyting to be imported. I am using just a subset of what we want to support in PKSim
That way we also get fraction error if we wish